src/AccessibilityTranslator.js

```diff
@@ -85,6 +85,14 @@ export default class AccessibilityTranslator extends React.Component {
       posParentSpace.y = (posGlobalSpace.y - parentBoundsInGlobalSpace.y) * (1 / displayObject.stage.scaleY);
       posParentSpace.width = (lowerRight.x - posGlobalSpace.x) * (1 / displayObject.stage.scaleX);
       posParentSpace.height = (lowerRight.y - posGlobalSpace.y) * (1 / displayObject.stage.scaleY);
+      if (posParentSpace.width < 0) {
+        posParentSpace.width = -posParentSpace.width;
```

Comment:
maybe just a style thing, but wouldn't it make sense to always [abs](https://www.w3schools.com/jsref/jsref_abs.asp) the width & height if we never want it to be negative?

Reply:
If it was unknown if it was negative or not, then I would agree and use abs.  In this case, it is known due to the if statement which is needed for both making sure the value is positive and adjusting the position.